### ISSUE TITLE
payment issue into my account

### ISSUE THREAD

**Issue by jbizzy42** [bitpay/copay#9038]
I sent $500 to my account from my paxful wallet and it shows sent there but I still haven't got it yet

here is my copay address 1Bi9cA197GLkRvun56h3YX8zAW2mxhAdKV

**Comment by jamesscookk** [comment]
2 hours

**Comment by BGalliotti** [comment]
Hello @jbizzy42 
Thanks for reaching out to us.

**Please open up your BitPay app and try the following steps:**

- Settings > Wallet > More Options > Transaction History > Clear cache
- Settings > Wallet > More Options > Wallet Addresses > Scan addresses for funds

Please let us know if you have any additional questions.
Kind regards.

**Comment by am05mhz** [comment]
I'm having similar issue, this moring i withdraw my bitcoin from mining pool, but till now i still have not receive it

My wallet address: 1BFMhfjqqF49wqzzu7EkLyeGLP8rpTMokf

**Comment by BGalliotti** [comment]
Hello @am05mhz 

Thanks for reaching out to us.

We are sorry you have had this bad experience.

Here is your wallet report:

```
* Balance
  Total: 0.00012476 btc (0.00000000 btc locked)
  Confirmed: 0.00005065 btc (0.00000000 btc locked)
  Available: 0.00012476 btc (0.00005065 btc confirmed / 0.00007411 btc unconfirmed)
  By address:
    1BFMhfjqqF49wqzzu7EkLyeGLP8rpTMokf (m/0/0): 0.00012476 btc

* No pending tx proposals.

* Send max
  Maximum spendable amount at "normal" fee level (0.00016656 btc per kb): 0.00005880 btc
  Available UTXOs:
    0.00007411 btc unconfirmed
    0.00005065 btc 1357 confirmations

* Main addresses (not including change addresses)
  [Sun Dec 10 2017 16:07:53 GMT-0500] 1BFMhfjqqF49wqzzu7EkLyeGLP8rpTMokf (m/0/0)
  [Wed Aug 15 2018 19:20:10 GMT-0400] 13CEyg1WQ7K2W9Vr47XWsMa2CWZEjsg2AM (m/0/1)
  [Wed Aug 15 2018 19:27:00 GMT-0400] 1G2vwVbn61JL9d3rcPcde52ymVvJcJWQhx (m/0/2)
  [Mon Sep 03 2018 20:59:39 GMT-0400] 12icKijX2cfH65pZfHDWQit3GPCsAqr52W (m/0/3)

* Tx history
  [Tue Sep 04 2018 11:48:25 GMT-0400] 87a1992ac343db6b330266050052496040273b27602759798d8b592af9573c84 <= received 0.00007411 btc (fee/kb 0.00001850 btc) (0 confirmations)
  [Sun Aug 26 2018 12:54:44 GMT-0400] 8c535ba58a8d34db4b6e8c7ba091590d3b5351d2a9cb9fc5248532d6e8027692 <= received 0.00005065 btc (fee/kb 0.00001001 btc) (1357 confirmations)

```
**Can you see this in your wallet transactions?**

Feel free to get back to us in case you have any questions or concerns.

**Comment by am05mhz** [comment]
Hi,

Now I can see the transaction, but the transaction is still in confirmation status.

Thanks

Amos

On Sep 4, 2018 22:51, BGalliotti <notifications@github.com> wrote:

Hello @am05mhz<https://eur04.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2Fam05mhz&data=02%7C01%7C%7C134c899a81204356536008d6127e42be%7C84df9e7fe9f640afb435aaaaaaaaaaaa%7C1%7C0%7C636716730820808588&sdata=CFQP4KnPGDHd7ynfVfqNA36JUCegMw6B5PADxXjT6%2F8%3D&reserved=0>

Thanks for reaching out to us.

We are sorry you have had this bad experience.

Here is your wallet report:

* Balance
  Total: 0.00012476 btc (0.00000000 btc locked)
  Confirmed: 0.00005065 btc (0.00000000 btc locked)
  Available: 0.00012476 btc (0.00005065 btc confirmed / 0.00007411 btc unconfirmed)
  By address:
    1BFMhfjqqF49wqzzu7EkLyeGLP8rpTMokf (m/0/0): 0.00012476 btc

* No pending tx proposals.

* Send max
  Maximum spendable amount at "normal" fee level (0.00016656 btc per kb): 0.00005880 btc
  Available UTXOs:
    0.00007411 btc unconfirmed
    0.00005065 btc 1357 confirmations

* Main addresses (not including change addresses)
  [Sun Dec 10 2017 16:07:53 GMT-0500] 1BFMhfjqqF49wqzzu7EkLyeGLP8rpTMokf (m/0/0)
  [Wed Aug 15 2018 19:20:10 GMT-0400] 13CEyg1WQ7K2W9Vr47XWsMa2CWZEjsg2AM (m/0/1)
  [Wed Aug 15 2018 19:27:00 GMT-0400] 1G2vwVbn61JL9d3rcPcde52ymVvJcJWQhx (m/0/2)
  [Mon Sep 03 2018 20:59:39 GMT-0400] 12icKijX2cfH65pZfHDWQit3GPCsAqr52W (m/0/3)

* Tx history
  [Tue Sep 04 2018 11:48:25 GMT-0400] 87a1992ac343db6b330266050052496040273b27602759798d8b592af9573c84 <= received 0.00007411 btc (fee/kb 0.00001850 btc) (0 confirmations)
  [Sun Aug 26 2018 12:54:44 GMT-0400] 8c535ba58a8d34db4b6e8c7ba091590d3b5351d2a9cb9fc5248532d6e8027692 <= received 0.00005065 btc (fee/kb 0.00001001 btc) (1357 confirmations)



Can you see this in your wallet transactions?

Feel free to get back to us in case you have any questions or concerns.

**Comment by BGalliotti** [comment]
Hello @am05mhz 
Thanks for reaching out to us.

We are glad to know you can see the transactions.

If the transaction is still pending confirmations, you can only wait for that. 

If you want to spend your funds you can try the following:

- Open the wallet app
- Go to **Advanced**
- Enable **Use unconfirmed funds**

**Bear in mind this will not work for invoice payments and giftcard purchases and card top-ups.**

**You can read more information about transaction delays here: #8712**

Please let us know if you have any additional questions.
Kind regards.
